exporter/datadogexporter/internal/metadata/internal/gohai/processes_other.go

```diff
@@ -0,0 +1,25 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build !linux && !darwin
+// +build !linux,!darwin
+
+package gohai // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/metadata/internal/gohai"
+
+import "go.uber.org/zap"
+
+// GetProcessesPayload currently just a stub.
+func NewProcessesPayload(hostname string, logger *zap.Logger) *ProcessesPayload {
+	// unimplemented for misc platforms.
+	return &ProcessesPayload{}
```

Comment:
But this is still the same payload, so what will happen once it start being used?

Reply:
we would send empty json the backend .

Reply:
Got it, thanks for clarifying. Resolving.